tests/manager/manager-executeWriteCommand_error-003.phpt

```diff
@@ -0,0 +1,36 @@
+--TEST--
+MongoDB\Driver\Manager::executeWriteCommand() throws CommandException for unsupported update operator
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php NEEDS('STANDALONE'); CLEANUP(STANDALONE); ?>
```

Comment:
This shouldn't need `CLEANUP()`, since the write never completes successfully and no data will be left behind in the collection.